app/Lib/SystemTag.php

```diff
@@ -84,10 +83,11 @@ static function intToTag(int $int): string {
      * @return int
      */
     static function tagToInt(string $tag): int {
```

Comment:
I would change this function completely.. it still assumes 2 chars.. would also change the intToTag to match once you change it..

Reply:
I looked into it, but not sure if it's worth the hustle. Two letters is 260 or smth variations of sigs for one class. I'd rather get punched in the dick than scan this many sigs